package.json

```diff
@@ -10,26 +10,9 @@
     "type": "git",
     "url": "https://github.com/MetaMask/test-dapp-multichain.git"
   },
-  "sideEffects": false,
-  "type": "module",
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./dist/index.d.mts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
-        "types": "./dist/index.d.cts",
-        "default": "./dist/index.cjs"
-      }
-    },
-    "./package.json": "./package.json"
-  },
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.cts",
+  "main": "index.js",
   "files": [
-    "dist"
+    "dist/"
```

Comment:
Hmm, I think this works the same with or without the slash, but this shouldn't hurt at least